drivers/watchdog/wdt_fwdgt_gd32.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2021 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT gd_gd32_free_watchdog
```

Comment:
compatible should use GD terminology: fwdgt

Reply:
Fixed it.
free_watchdog -> fwdgt
window_watchdog -> wwdgt